catalyst/contrib/datasets/__init__.py

```diff
@@ -1,4 +1,20 @@
 # flake8: noqa
```

Comment:
let's remove noqa :)

Reply:
we can't do this because of `unused imports` error